components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/OpenIDConnectConfiguration.java

```diff
@@ -375,7 +376,26 @@ public OpenIDConnectConfiguration addScopeValidatorsItem(String scopeValidatorsI
         return this;
     }
 
+        /**
```

Comment:
Shall we fix the alignment here?

Reply:
This code part is no more used in the master.